samples/client/petstore/swift5/alamofireLibrary/PetstoreClient/Classes/OpenAPIs/Models/AdditionalPropertiesClass.swift

```diff
@@ -12,7 +12,7 @@ public struct AdditionalPropertiesClass: Codable {
     public var mapString: [String: String]?
     public var mapMapString: [String: [String: String]]?
 
-    public init(mapString: [String: String]?, mapMapString: [String: [String: String]]?) {
+    public init(mapString: [String: String]? = nil, mapMapString: [String: [String: String]]? = nil) {
```

Comment:
@Allezxandre looks like many parameters are default to `nil`. Is that expected?

Reply:
This is not related to this PR, it was introduced in another one, but apparently the samples were outdated.

Reply:
I actually tested with the master but didn't get these `nil` default value. Maybe my test wasn't done right.

Reply:
Let me test on master

Reply:
When I run `./mvnw package && ./bin/generate-samples.sh bin/configs/other/swift5-*` on the master, I get all of those `nil` changes

Reply:
Thanks for confirming that. 

Is the `nil` default value expected? If not, we can remove it later in another PR. ( I think I know where to fix it )

Reply:
Yes it is expected. 
Basically this is just providing default `nil` values for for nullable types to shorten the constructor to the library user.